IdentityCore/src/webview/systemWebview/ios/MSIDSFAuthenticationSessionHandler.m

```diff
@@ -66,10 +66,12 @@ - (void)startWithCompletionHandler:(MSIDWebUICompletionHandler)completionHandler
         {
             NSError *cancelledError = MSIDCreateError(MSIDErrorDomain, MSIDErrorUserCancel, @"User cancelled the authorization session.", nil, nil, nil, nil, nil, YES);
             
+            self.webAuthSession = nil;
             if (completionHandler) completionHandler(nil, cancelledError);
             return;
         }
         
+        self.webAuthSession = nil;
         completionHandler(callbackURL, authError);
```

Comment:
Do we want to nil out `webAuthSession` before line 62 too?

Reply:
We should. Addressed.